fakecam/fake.py

```diff
@@ -244,14 +248,18 @@ async def mask_frame(self, session, frame):
                 print(f"Mask request failed, retrying: {e}")
                 traceback.print_exc()
 
+        foreground_frame = background_frame = frame
```

Comment:
While I think it's really good that you have renamed the various layers, maybe take a moment to thing about it?

Having the name `frame` for the output, makes sense, as ultimately, that is what we are generating

The background, is also quite obvious, `background` makes sense. Even the `foreground` frame is still sensible, for the 'podium' as we call it here.

But the 'inbetween frame' wouldn't that be something like 'layer1_frame', 'webcam_frame' or 'actor_frame'? This would make adding additional layers in the future far more easier, so your stack would end up looking something like this:

frame = {background + layerZ + layerY + layerX + actor + foreground)

what do you think?